src/components/SkipLink/SkipLink.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import cx from 'classnames';
+
+type SkipLinkProps = {
+  className?: string;
+  href: string;
+  isNarrow?: boolean;
+  skipLinkText: string;
+};
+
+export const SkipLink = ({
+  className,
+  href,
+  isNarrow,
+  skipLinkText
+}: SkipLinkProps) => {
+  const classNames = {
+    wrapper: cx('cc-skip-link', {
+      [className]: className
+    }),
+    link: cx('cc-skip-link__link', {
+      'cc-skip-link__link--narrow': isNarrow
```

Comment:
`isNarrow` is a little confusing as a property name. If this controls how the link is positioned (absolute or static) how about something more in line with that and maybe even make it enumerable for ease of expansion in future?

type
```
position: 'static' | 'float';
```

prop
```
position = static
```

classes
```
const classNames = {
    wrapper: cx('cc-skip-link', {
      [className]: className
    }),
    link: cx('cc-skip-link__link', {
      'cc-skip-link__link--float': position === 'float'
  })
};
```